ui/ui-components/pages/model/[modelId]/source/[sourceId]/schedule.tsx

```diff
@@ -488,7 +492,7 @@ export default function Page(props) {
                                             <option
                                               key={`op-opt-${localFilter.key}-${op}`}
                                             >
-                                              {op}
+                                              {filterOptionDescriptions[op]}
```

Comment:
I think this will change how this form works. In the `onChange()` handler, we are using `e.target.value`, which is now the human-readable name from the dictionary.  I think the option also needs `value=${op}`